library/Icingadb/Widget/Detail/QuickActions.php

```diff
@@ -71,7 +71,10 @@ protected function assemble()
             );
         }
 
-        if ($this->isGrantedOn('icingadb/command/send-custom-notification', $this->object)) {
+        if (
+            $this->isGrantedOn('icingadb/command/send-custom-notification', $this->object)
+            && $this->object->notifications_enabled //TODO: not sure, Model contains a forced switch
```

Comment:
Yeah, and because of that switch this is nonsense. The tooltip of the option clearly states this.